filebeat/docs/modules-dev-guide.asciidoc

```diff
@@ -0,0 +1,304 @@
+[[filebeat-modules-devguide]]
+== Developer guide: creating a new module
+
+This guide will walk you through creating a new Filebeat module.
+
+All Filebeat modules currently live in the main
+https://github.com/elastic/beats[Beats] repository. To clone the repository and
+build Filebeat (which you will need for testing), please follow the general
+Beats https://github.com/elastic/beats/blob/master/CONTRIBUTING.md[CONTRIBUTING]
```

Comment:
Not only related to this guide, but we should probably have contributing as part of the docs to also have it specific to a version.